src/index.html

```diff
@@ -13,6 +13,131 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div
+        class="calendar__day"
+        data-day="1"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="2"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="3"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="4"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="5"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="6"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="7"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="8"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="9"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="10"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="11"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="12"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="13"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="14"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="15"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="16"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="17"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="18"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="19"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="20"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="21"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="22"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="23"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="24"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="25"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="26"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="27"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="28"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="29"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="30"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="31"
+      ></div>
```

Comment:
The calendar days are hardcoded. This is not a scalable solution. Consider using a scripting language like JavaScript to dynamically generate the calendar days based on the month length and the start day.